cloudify_agent/operations.py

```diff
@@ -48,6 +49,11 @@
 
 @operation
 def install_plugins(plugins, **_):
+    tenant_mask = False
+    if ctx.tenant_name != workflow_ctx.tenant_name:
```

Comment:
Can you explain this scenario?
Isn't it possible to propagate the tenant information?
ctx._context is internal, its modification should be somehow encapsulated...